src/lib.rs

```diff
@@ -136,14 +136,14 @@ fn progress(
 ) -> Result<(), failure::Error> {
     let template = template::substitute(name, force)?;
 
-    let pbar = progressbar::new();
-    pbar.tick();
-
     let mut config_path = dir.clone();
     config_path.push(CONFIG_FILE_NAME);
 
     let template_config = Config::new(config_path)?.map(|c| c.template);
 
+    let pbar = progressbar::new();
```

Comment:
put this after the template config since it swallows stdout, and I don't want to pass pbar to config constructor